### PR TITLE
New API to Fetch WhiteListed Domains for email - KB-5850

### PR DESCRIPTION
**Pull request by sreeragksgh**

### CODE REVIEW

**Comment by sreeragksgh** [comment]
Jira Ticket : https://karmayogibharat.atlassian.net/browse/KB-5850

File: src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java [hunk 1]
```diff
@@ -588,4 +621,29 @@ private Boolean isApprovedDomains(String emailDomain, String domainType) {
 				Constants.KEYSPACE_SUNBIRD, Constants.TABLE_MASTER_DATA, propertyMap, Arrays.asList(Constants.CONTEXT_TYPE, Constants.CONTEXT_NAME));
 		return CollectionUtils.isNotEmpty(listOfDomains);
 	}
+
+	private List<String> getApprovedDomainsFromDB() {
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java]: what will happen if the domain exist in both list?

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java (hunk 1 above)
Comment (sreeragksgh) [reply src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java]: Updated code to tackle this

File: src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java [hunk 2]
```diff
@@ -305,6 +306,38 @@ public void initiateCreateUserFlow(String registrationCode) {
 		}
 	}
 
+	public SBApiResponse getApprovedDomains() {
+		SBApiResponse response = createDefaultResponse(Constants.API_APPROVED_DOMAINS);
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/user/registration/service/UserRegistrationServiceImpl.java]: I thought this method adds all the necessary information into the response... 

if not -- just use ProjectUtil.createDefaultResponse

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

**Pull request merged**